spec/javascripts/unit/admin/products/services/unit_prices_spec.js.coffee

```diff
@@ -0,0 +1,102 @@
+describe "UnitPrices service", ->
```

Comment:
Let me know if you need help understanding imperial units so you can test them ;)

Reply:
I must confess that it makes me a little desperate, and that I will need someone to cheer me up ;)

Reply:
It seems that, in the shopfront, this is accurate:
<img width="1046" alt="Capture d’écran 2021-03-08 à 14 11 01" src="https://user-images.githubusercontent.com/296452/110325897-39aad700-8018-11eb-9dd3-3a02f57e09f6.png">

Thanks @andrewpbrett ! 🙏

Reply:
I've added some tests ;)